### DISCUSSION TITLE
New Idea - headlessui 2.0 for vue

### DISCUSSION THREAD

**Discussion by ngl5000** [tailwindlabs/headlessui#3213] [Ideas]
because, Vue is great and people use it..a lot

**Comment by anshulbansal02** [comment]
Gotta agree.

**Comment by damevin** [comment]
🙏 totally agree, it will be great

**Comment by Bosphoramus** [comment]
Yes, please 🥲

**Comment by LadyAiss** [comment]
Hello,

I have to up this one up because a lot of needed functionalities is lacking. For instance, exposing the `open` function alongside the `close` one would allow us to have a better handling of state management, which will be great for different toggling mechanisms and in the case of menus for instance, having a hierarchy.

Vue is under the top 3 frameworks, so I believe it should have first grade support.

**Comment by hemju** [comment]
I understand that React is the big one in the ecosystem, but at least the top frameworks should be supported.

Could we get information on whether there will ever be a version 2 for Vue? If yes, is there a timeline? That has implications for the production use of this library.

**Comment by lebowska** [comment]
I'll paste what was already reported on the other vue-related discussion, it seems that Adam has other priorities atm:

> Hey! No concrete plans to do Catalyst for Vue but not something I'm actually against, just hasn't earned a spot on the roadmap. Headless UI v2 for Vue I just don't have a good answer for — I think we should do it but our plate is already full for the rest of the year with Tailwind CSS v4.0 related projects so I just don't really know yet what we're going to do.

[Link](https://x.com/adamwathan/status/1807502683189772362)

**Comment by zangetsu02** [comment]
Hello, 
have there been any updates by any chance?
